assets/js/projects.js

```diff
@@ -30,6 +30,23 @@ document.addEventListener('DOMContentLoaded', function() {
             // Card data goes here
             // Example:
             // category: 'consultancy', 'software development', 'support and training'],
+            {
+              category: ['support and training'],
+              status: 'past',
+              imageUrl: 'assets/img/projects/eurelectri.jpeg',
+              title: 'Comprehensive PyPSA Training Session',
+              subtitle: 'Equipping energy enthusiasts with essential skills for open energy modeling',
+              description: 
+              'In December 2024, an in-depth PyPSA training session equipped participants with essential \
+              energy modeling skills. The training included five sessions: basic PyPSA concepts, installation \
```

Comment:
I'd probably highlight once again that the session has been quite an advanced one:

```suggestion
              'In December 2024, an in-depth PyPSA training session equipped participants with essential \
              advanced energy modeling skills. The training included five sessions: basic PyPSA concepts, installation \
```